payments-core/src/main/java/com/stripe/android/payments/core/injection/PaymentCommonModule.kt

```diff
@@ -128,5 +128,17 @@ internal abstract class PaymentCommonModule {
                 is SetupIntentClientSecret -> setupIntentFlowResultProcessor
             }
         }
+
+        @Provides
+        @Singleton
+        fun provideAnalyticsRequestFactory(
+            context: Context,
+            lazyPaymentConfiguration: Lazy<PaymentConfiguration>,
+            defaultProductUsageTokens: Set<String>,
+        ) = AnalyticsRequestFactory(
+            context,
+            { lazyPaymentConfiguration.get().publishableKey },
+            defaultProductUsageTokens,
+        )
```

Comment:
This could be an `@Inject` constructor but that would require making it public. I think it's better to provide through the module and keep it internal.